### PR TITLE
[admin] Add docker compose to installation section

### PR DESCRIPTION
**Pull request by istarkov**
ref https://github.com/webstudio-is/webstudio-designer/pull/371

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -20,9 +20,10 @@ If you want to contribute and don't know where to start, here is a step-by-step
 
 1. Install [Node.js](https://nodejs.dev/learn/how-to-install-nodejs)
 2. Install [Yarn](https://yarnpkg.com/) `npm i -g yarn`
-3. Install Postgres, in here there are two main options:
-    1. Install the [official Postgres app](https://www.postgresql.org/download/) - Make sure to remember the username and password you choose since you will need it for the connection string
-    2. On a mac you can also install [Postgres.app](https://postgresapp.com/) and in that case you will have no password and your username will be the same as your system username.
+3. Intall [Docker Engine](https://docs.docker.com/engine/install/), run `docker compose -f ./apps/designer/ docker-compose.yaml up -d` to start Postgres _(use PGPORT environment variable in case of default is already used)_.
+   Or install Postgres, in here there are two main options:
```
Comment (rpominov) [inline docs/contributing.md:24]: I'd probably keep the main label of this step "Install Postgres" and would put "Install Docker" as one of the ways of installing Postgres.

Also, typo: Intall -> Install

Comment (kof) [reply docs/contributing.md:24]: definitely keep the installation without docker

Comment (kof) [reply docs/contributing.md:24]: I would just make docker the 3rd option

File: docs/contributing.md [hunk 2]
```diff
@@ -20,9 +20,10 @@ If you want to contribute and don't know where to start, here is a step-by-step
 
 1. Install [Node.js](https://nodejs.dev/learn/how-to-install-nodejs)
 2. Install [Yarn](https://yarnpkg.com/) `npm i -g yarn`
```
Comment (TrySound) [inline docs/contributing.md:22]: Could you please change here
```
Enable [Corepack](https://nodejs.org/api/corepack.html) `corepack enable`
```

Comment (kof) [reply docs/contributing.md:22]: ohh that doesn't exist in node version we are using, that means its currently useless

Comment (kof) [reply docs/contributing.md:22]: and it has experimental status

Comment (istarkov) [reply docs/contributing.md:22]: For volta users it needs also `volta install corepack` ;-)

**Review by rpominov — APPROVED** [review]

**Pull request merged**